react/react2/week1/github-search/src/components/InputBar.js

```diff
@@ -0,0 +1,22 @@
+import { useContext } from "react";
+import { SearchContext } from "./SearchContext";
+
+export const InputBar = () => {
+  const contextValue = useContext(SearchContext);
+
+  const inputHandler = (e) => {
+    // console.log("Input handler works");
+    // console.log(contextValue.inputValue);
+    contextValue.setInputValue(e.target.value);
```

Comment:
`setInputValue `is neither defined anywhere in this file nor passed as input to this function!

Reply:
it is defined in the App.js file and passed in contextValues (they are defined - same file, line 47)